### ISSUE TITLE
Sudden Compatibilty Issue With Sudden Compatibility Release

### ISSUE THREAD

**Issue by leonid-s-usov** [Alamofire/AlamofireImage#349]
* [ ] I've read, understood, and done my best to follow the [*CONTRIBUTING guidelines](https://github.com/Alamofire/AlamofireImage/blob/master/CONTRIBUTING.md).

## What did you do?

updated carthage

## What did you expect to happen?

build succeed

## What happened instead?

build has failed

## Alamofire Environment

**Alamofire version:**
4.8.2
**Xcode version:**
9.4.1
**Swift version:**
4.1
**Platform(s) running AlamofireImage:**
iOS, macOS
**macOS version running Xcode:**
10.13.6

```
/Users/.../Carthage/Checkouts/AlamofireImage/Source/ImageDownloader.swift:462:44: 
error: value of type '[(receiptID: String, filter: ImageFilter?, completion: ImageDownloader.CompletionHandler?)]' 
(aka 'Array<(receiptID: String, filter: Optional<ImageFilter>, completion: Optional<(DataResponse<UIImage>) -> ()>)>') 
has no member 'firstIndex'

            if let index = responseHandler.operations.firstIndex(where: { $0.receiptID == requestReceipt.receiptID }) {
                           ~~~~~~~~~~~~~~~~^~~~~~~~~~ ~~~~~~~~~~```

**Comment by jshier** [comment]
@leonid-s-usov If you want, give the branch in #350 a try to see if that solves your issue. We'll be rolling it into a release soon.

**Comment by jshier** [comment]
@leonid-s-usov 3.5.2 has shipped, please open a new issue if you have any other issues building. Thanks for the report and sorry about that! Cheers! 🍻

**Comment by leonid-s-usov** [comment]
Thanks! Checking

**Comment by leonid-s-usov** [comment]
Forgot to mention that it was solved, thanks.

It wasn't critical in my case, I had simply fixed the version to 3.5.1 in the Carthage and that served well as a workaround.
